class/catalog.class.php

```diff
@@ -836,7 +913,441 @@ public function GetFilecsv()
 			$response = '0,<p>Aucune mise à jour catalogue Ecopresto disponible.</p>';
 		return $response;
 	}
+	/**
+	 * Enregistre le catalogue CSV en provennance d'Ecopresto
+	 *
+	 *
+	 * @param aucun
+	 * @return false si erreur, nb ligne du fichier sinon
+	 */
+	public function GetCatalogCSV()
+	{
+		$domain = Configuration::get('PS_SHOP_DOMAIN');
+		
+		$cle = $this->tabConfig['ID_ECOPRESTO'];
+
+		$filename = $this->fichierDistant;
+		include_once _PS_ROOT_DIR_.'/modules/ecopresto/class/download.class.php';
+		$download = new DownloadBinaryFile();
+		$response = '';
+		
+		
+		if ($download->load($filename) == true)
+		{
+			$download->saveTo(_PS_ROOT_DIR_.'/modules/ecopresto/files/'.$this->fichierImport);

```

Comment:
We reuse old piece of code and don't change this part from previous version (even if some variable were use, it was always 'catalog.csv'. 
Currently, we are rewriting the generation process for this file from our server. 
I notice your remark and i'm agree with you, but we don't have time to finish it right now. So this part will be change in a future version.